packages/vue/src/components/organisms/SfHeader/SfHeader.vue

```diff
@@ -1,115 +1,135 @@
 <template>
-  <div class="sf-header" :class="{ 'is-sticky': sticky, 'is-hidden': hidden }">
-    <div class="sf-header__wrapper">
-      <header ref="header" class="sf-header__header">
-        <!--@slot Use this slot to replace logo with text or image-->
-        <slot name="logo" v-bind="{ logo, title }">
-          <SfLink link="/">
-            <SfImage
-              v-if="logo"
-              :src="logo"
-              :alt="title"
-              class="sf-header__logo"
+  <header
+    ref="header"
+    v-click-outside="closeHandler"
+    class="sf-header sf-header__wrapper sf-header__header"
+  >
+    <!--@slot Use this slot to replace logo with text or image-->
+    <slot name="logo" v-bind="{ logo, title }">
+      <SfLink link="/">
+        <SfImage v-if="logo" :src="logo" :alt="title" class="sf-header__logo" />
+        <h1 v-else class="sf-header__title">{{ title }}</h1>
+      </SfLink>
+    </slot>
+    <div class="sf-header__aside">
+      <!-- @slot Use this slot for language or currency selector -->
+      <slot name="aside" />
+    </div>
+    <div class="sf-header__actions">
+      <nav class="sf-header__navigation">
+        <slot name="navigation">
+          <div v-if="!isVisibleOnMobile" class="sf-header__menu">
+            <SfButton
+              v-for="(item, i) in menuItems"
+              :key="i"
+              class="sf-header__menu-item sf-header__menu-item--desktop sf-button--pure"
+              v-on="$listeners"
+              @mouseenter="$emit('mouseenter:button', item)"
+              @mouseleave="$emit('mouseleave:button', '')"
+              @click="$emit('click:button', item)"
+            >
+              {{ item }}
+            </SfButton>
+          </div>
+          <SfSidebar
+            :visible="isVisibleOnMobile && openSidebar"
+            :persistent="true"
+            :overlay="false"
+            class="sf-header__menu--sidebar"
+            @close="$emit('close')"
+          >
+            <SfMenuItem
+              v-for="(item, i) in menuItems"
+              :key="i"
+              :label="item"
+              class="sf-header__menu-item sf-header__menu-item--mobile"
+              @click="$emit('click:button', item)"
             />
-            <h1 v-else class="sf-header__title">{{ title }}</h1>
-          </SfLink>
+          </SfSidebar>
+          <div v-if="openContent" class="sf-header__menu-content">
+            <template v-for="item in menuItems">
```

Comment:
No `:key` attribute?

Reply:
I think it's not necessary for slot. I based on this article https://michaelnthiessen.com/supercharge-your-slots/#adding-slots-dynamically.